### PR TITLE
Anticipate the Governor's announcement of teachers

### PR DESCRIPTION
**Pull request by johnhawkinson**
Anticipate the Governor's announcement of limited Phase 2 Group 3 for teachers

    Also add the rest of P2G3 in a comment in case he opens it up to Transit/grocery/foodworkers.
Screen Shot 2021-03-03 at 09 22 28

### CODE REVIEW

File: src/components/StateEligibility.js [hunk 1]
```diff
@@ -48,6 +48,26 @@ export default function StateEligibility() {
                 </AccordionSummary>
                 <AccordionDetails className={classes.accordionDetails}>
                     <List>
+                        <EligibilityGroupItem>
+                            <a
+                                href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases"
```
Comment (tkjones117) [inline src/components/StateEligibility.js:53]: Eventually it'd be nice to point this to the equivalent education-specific link (like how below we link to info for each specific group) but until we know that url, this is a good placeholder.

**Comment by tkjones117** [comment]
Waiting to approve until we get word from Baker.

File: src/components/StateEligibility.js (hunk 1 above)
Comment (johnhawkinson) [reply src/components/StateEligibility.js:53]: Yes, that's the intent!

**Comment by johnhawkinson** [comment]
Eligibility won't happen until March 11, this can wait.
https://twitter.com/MattPStout/status/1367127174596210689

**Pull request closed without merging**